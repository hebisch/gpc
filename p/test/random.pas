program random(output);

uses GPC;

const
Size = 110;
tries = 10000;

type
density=array[0..Size] of Integer;

var
den :density;
Count :Integer;
loc :Integer;

begin
SeedRandom(42);
for Count:=0 to Size do
        den[Count] := 0;
for Count:=1 to tries do begin
        loc := round(random * (Size - 10));
        den[loc] := den[loc] + 1;
        end;
for Count:=0 to Size div 10 - 1 do begin
        for loc:=10 * Count to 10 * Count + 9 do
                Write(den[loc]:7);
        WriteLn;
        end;
end.
