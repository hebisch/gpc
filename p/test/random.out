     47    114     99     97    106    103     98    111    111    105
     85    101    107    106     98     86     95    105     95     85
    101     94    100    118     93    108     83    106     99     74
    102    103     93     98    100    104     96     92     99    101
    127    105     93    101     94    102     96     91    104     83
     94     98    104     96    108    115     99     92    116    102
    124     96    114    106     95    100    100    105    106     95
     97     99     86    114    102    106     99     89    101     96
    107     97    119    111    107     87    102    105    105     85
     98     95     72     99    110     91     91    101    103    102
     45      0      0      0      0      0      0      0      0      0
